server.go

```diff
@@ -1265,6 +1268,29 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		if err != nil {
 			return nil, err
 		}
+
+		// Copy the policy for legacy channels and set the blob flag
+		// signalling support for anchor channels.
+		anchorPolicy := policy
+		anchorPolicy.TxPolicy.BlobType |=
+			blob.Type(blob.FlagAnchorChannel)
+
+		s.anchorTowerClient, err = wtclient.New(&wtclient.Config{
```

Comment:
what is the reason we need it to be separate?

Reply:
Currently each "client" has a full pipeline that is capable of processing backups for a specific channel type, each maintains its own set tower candidates/state machines/session queues/etc. The option was either to multiplex _inside_ the client, but that seemed pretty invasive and more risky given that the full flow operates pretty reliably today. We decided to go with the approach of just spinning up two separate clients so that each is fully isolated and we know their wont be any unexpected bugs introduced there.

Down the road, the goal would be to add another layer of multiplexing between the links and the "clients", which facilitates sending channel updates to the correct pipelines. For now since we only have two, it is easier just to expose the multiplexing in the link/peer. However, as we start to add more channel types, e.g. anchor-zero-fee-htlc, anchor-taproot, etc, we plan on circling back so that lnd just sees one unified megaclient that handles everything under the hood.